### PR TITLE
Fixed missing link

### PR DESCRIPTION
**Pull request by Krzmbrzl**

### CODE REVIEW

File: hugo/content/contribute.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Mumble and all around it needs documentation. Great documentation for a great pr
 
 **Website**: This website has a public hugo source repository at [mumble-www](https://github.com/mumble-voip/mumble-www). You can report [issues](https://github.com/mumble-voip/mumble-www/issues), create [requests](https://github.com/mumble-voip/mumble-www/issues), and create pull requests for implemented change suggestions.
 
-**Wiki**: Our [documentation wiki] covers some more advanced topics than this website; for example for server administrators and contributors. Any contributions welcome, be it improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
+**Wiki**: Our [documentation wiki](https://wiki.mumble.info/wiki/Main_Page) covers some more advanced topics than this website; for example for server administrators and contributors. Any contributions welcome, be it improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
```
Comment (davidebeatrici) [inline hugo/content/contribute.md]:
```suggestion
**Wiki**: Our [documentation wiki](https://wiki.mumble.info) covers some more advanced topics than this website; for example for server administrators and contributors. Any contributions welcome, be it improving existing pages or creating new ones with useful content. Also see the {{< wiki "Category:Please_Improve" "Please Improve Category" />}}.
```

**Comment by davidebeatrici** [comment]
Could you squash the commits, please?

**Comment by Krzmbrzl** [comment]
Uhm... As soon as I found out how to do that, sure xD

**Comment by Krzmbrzl** [comment]
According to https://github.blog/2016-04-01-squash-your-commits/ squashing should be an option when merging the PR.

If not I'll do it tomorrow when I'm back at my PC

**Comment by davidebeatrici** [comment]
1. `git rebase -i HEAD~2`
2. Change `pick` to `squash` or `s` in the second commit's line, save and exit from the editor.
3. Change the new commit's message, save and exit from the editor.

**Comment by Kissaki** [comment]
Thank you

I formatted the commit message to be present tense (as is usually used and we do as well), added a qualfying prefix to the title, and improve the message content a bit.

**Pull request merged**